lib/library.go

```diff
@@ -93,6 +93,41 @@ func ExtractIdentityFromContactCode(bundleString *C.char) *C.char {
 	return C.CString(string(data))
 }
 
+// VerifySignatures ensure the signature pairs is valid
+//export VerifySignatures
+func VerifySignatures(signaturePairsStr *C.char) *C.char {
+	var signaturePairs [][]string
+
+	if err := json.Unmarshal([]byte(C.GoString(signaturePairsStr)), &signaturePairs); err != nil {
+		return makeJSONResponse(err)
+	}
+
+	if err := statusBackend.VerifySignatures(signaturePairs); err != nil {
+		return makeJSONResponse(err)
+	}
+
+	return nil
+}
+
+// Sign signs a string
+//export Sign
+func Sign(content *C.char) *C.char {
+	signature, err := statusBackend.Sign(C.GoString(content))
+
+	if err != nil {
+		return makeJSONResponse(err)
+	}
+
+	data, err := json.Marshal(struct {
+		Signature string `json:"signature"`
```

Comment:
do you want it to be a struct? json.Marshal(signature)?

Reply:
I would like it to be:
`{"signature": "tha"} `, so it's consistent with the error struct `{"error": "bah"}`, is this the right way to go about it?

Reply:
:+1:  it is probably easier to unmarshal if error and regular object is a struct